core/arch/arm/tee/entry_std.c

```diff
@@ -287,6 +288,8 @@ static void entry_invoke_command(struct thread_smc_args *smc_args,
 	struct tee_ta_session *s;
 	struct tee_ta_param param;
 
+	bm_timestamp();
```

Comment:
This should be a known ID on which REE gets comprehensive.
I think this should be like `bm_timestamp(CORE_PRE/POST_IMVOKE_TA);`
Maybe this can come later... Specified bm points and free bm points.